src/main/java/com/google/api/codegen/py/PythonSphinxCommentFixer.java

```diff
@@ -22,6 +22,8 @@
 
   /** Returns a Sphinx-formatted comment string. */
   public static String sphinxify(String comment) {
+    // escape '$' in the comment first
```

Comment:
This has nothing to do with Sphinix. This is java regex API's requirement, for example, https://github.com/googleapis/toolkit/blob/master/src/main/java/com/google/api/codegen/ruby/RDocCommentFixer.java#L64, The second argument of 'appendReplacement' cannot contain a non-escaped '$' in it.